src/main/kotlin/com/exactpro/th2/lwdataprovider/db/CradleMessageExtractor.kt

```diff
@@ -367,4 +369,47 @@ private fun StoredGroupedMessageBatch.toShortInfo(): String =
 data class CradleGroupRequest(
     val sort: Boolean,
     val preFilter: ((StoredMessage) -> Boolean)? = null,
-)
\ No newline at end of file
+)
+
+private enum class OrderStrategy {
+    DIRECT {
+        override fun checkBatchSequence(first: StoredGroupedMessageBatch, second: StoredGroupedMessageBatch): Boolean =
+            first.firstTimestamp <= second.firstTimestamp
+
+        override fun checkMessageSequence(message: StoredMessage?, batch: StoredGroupedMessageBatch): Boolean =
```

Comment:
Let's rename it to `checkMessageBeforeBatchTimestamp`

Reply:
`checkMessageBeforeBatchTimestamp` isn't correct name because
DIRECT: msg.timestamp < batch.firstTimestamp
REVERSE: msg.timestamp > batch.lastTimestamp

Reply:
Okay, then I would suggest using a name like `checkMessageInOrderWithBatch`